tb_plugin/fe/src/components/MemoryView.tsx

```diff
@@ -104,34 +177,81 @@ export const MemoryView: React.FC<IProps> = (props) => {
         <CardHeader title="Memory View" />
         <CardContent>
           <Grid direction="column" container spacing={1}>
-            <Grid item container direction="column" spacing={1}>
-              <Grid item>
-                <Grid container justify="space-around">
-                  <Grid item>
-                    <InputLabel id="memory-device">Device</InputLabel>
-                    <Select
-                      labelId="memory-device"
-                      value={device}
-                      onChange={onDeviceChanged}
-                    >
-                      {devices.map((device) => (
-                        <MenuItem value={device}>{device}</MenuItem>
-                      ))}
-                    </Select>
+            <Grid item>
```

Comment:
need we hide the Grid when memoryCurveGraph is nil?

Reply:
It can be done if curve rows is empty on first load. Let's me check how to hide it first....